docs/integrations/saas-cloud/proofpoint-tap.md

```diff
@@ -0,0 +1,182 @@
+| json "id","type","cluster","sender","recipient","messageParts[*].disposition","messageParts[*].sandboxStatus","messageSize","modulesRun","policyRoutes","senderIP","threatsInfoMap[*].classification","threatsInfoMap[*].threatType","threatsInfoMap[*].threatStatus","impostorScore","malwareScore","phishScore","spamScore","quarantineFolder","quarantineRule","subject" as id,type,cluster,sender,recipient,dispositions,sandboxStatuses,messageSize,modules_run,policy_route,sender_ip,threat_categories,threat_types,threat_status,impostor_score,malware_score,phish_score,spam_score,quarantine_folder,quarantine_rule,subject nodrop
+| extract field=threat_status "\"?(?<status>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=threat_types "\"?(?<threat_type>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=threat_categories "\"?(?<threat_category>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=modules_run "\"?(?<module>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| extract field=policy_route "\"?(?<policy>[\w\s\-&.,]*)\"?[,\n\]]" multi
+| where type matches "{{message_type}}"
+| where threat_category matches "{{threat_category}}"
+| where threat_type matches "{{threat_type}}"
+| where status matches "{{threat_status}}"
+| where module matches "{{module}}"
+| where policy matches "{{policy_route}}"
+| count_distinct(id)
+```
+
+```sql title="Sample Click Events Query (Number of Clicks)"
+_sourceCategory="milan_proofpoint_tap" ("CLICK_PERMITTED" or "CLICK_BLOCKED")
+| json "id","type","threatUrl","classification","clickIP","senderIP","sender","recipient","threatStatus" as id,type,threat_url,category,click_ip,sender_ip,sender,recipient,threat_status nodrop
+| where type matches "{{click_type}}"
+| where category matches "{{threat_category}}"
+| where threat_status matches "{{threat_status}}"
+| count_distinct(id)
+```
+
+## Set up collection
+
+To set up [Cloud to Cloud Integration Proofpoint TAP Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/proofpoint-tap-source/) for the Proofpoint TAP app, follow the instructions provided. These instructions will guide you through the process of creating a source using the Proofpoint TAP source category, which you will need to use when installing the app. By following these steps, you can ensure that your Proofpoint TAP app is properly integrated and configured to collect and analyze your Proofpoint TAP data.
+
+## Installing the Proofpoint TAP app​
+
+This section provides instructions on how to install the Proofpoint TAP app, as well as examples of each of the dashboards. The app pre-configured searches and dashboards provide easy-to-access visual insights into your data.
+
+{@import ../../reuse/apps/app-install.md}
+
+## Viewing Proofpoint TAP dashboards​​
+
+* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
```

Comment:
If we're showing filter and funnel icon in the dashboards, then we should link that here. Otherwise, this sentence is not correct.

Reply:
We could say as shown in the following dashboards.

Reply:
Have you added the change? I still see the same.

Reply:
Hi @JainM6 , you are correct. Because this snippet appears in ~50 different apps docs, I've created an Asana ticket for Jagadisha [here](https://app.asana.com/0/1183741177809740/1204771809032685/f) to tackle here once this PR goes out.

Reply:
@JainM6 I have changed it as "All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following dashboards.".
@kimsauce Thanks Kim, I will work on that.